### ISSUE TITLE
Change directory problems with micropython

### ISSUE THREAD

**Issue by asherevan** [thonny/thonny#3207]
After changing directories with os.chdir and then trying to change back to the previous directory with the open file dialog in thonny it only shows the contents of the same directory as folders.

**Comment by aivarannamaa** [comment]
Thank you for the report and sorry for the late reply!

> it only shows the contents of the same directory as folders.

I didn't quite understand this. Can you rephrase it?

**Comment by asherevan** [comment]
Instead of changing directories and displaying the files it just stays in the same directory and shows the files as folders (with no contents) instead.

**Comment by aivarannamaa** [comment]
I was not able to reproduce this. Could you please post a screenshot?

**Comment by asherevan** [comment]
I’m on a raspberry pi so I couldn’t take a screenshot, but here’s the best picture I could take.

This is after changing directories with os.chdir then trying to change to the home directory in the file browser. And it is only showing the contents of the other directory.

![AA1658D1-29D6-479C-9540-FC250958B8E8](https://github.com/user-attachments/assets/79609830-31eb-4e13-bb49-bb988a08726b)
